documentation/docs/evaluators/general_evaluator.md

```diff
@@ -1 +1,8 @@
-# General evaluator
\ No newline at end of file
+# General evaluator
+The general runner is capable of running the standard language of the following languages
```

Comment:
what do you mean with the standard language?

Reply:
The language without any extra installations

Reply:
Maybe base language better?

Reply:
https://github.com/SELab-2/UGent-3/pull/424/commits/691b8301650b9c917c2b1a5a893714ae1d2ff1f6